README.md

```diff
@@ -1,2 +1,28 @@
-# photon-extension-kit
-Photon styles for your brand new extension
+# Photon Extension Kit
+
+Photon styles for your brand new extension.
```

Comment:
Yeah, this is a good idea. I will create an `/examples` folder and inside we could have a couple of different examples, the first of which is `/components`! I know that @designakt is collecting some extensions and he is identifying what are the most used components and layouts so that we could create examples boilerplates.

Reply:
See da2af69 ☺️